text/0000-fix-fees.md

```diff
@@ -0,0 +1,113 @@
+- Title: fix-fees
+- Authors: [John Tromp](mailto:john.tromp@gmail.com)
+- Start date: August 23, 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000) 
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+Change Grin's minimum relay fees to be weight proportional and make output creation cost about a Grin-cent.
+Use least significant bits in fee to specify desired tx priority.
+
+# Motivation
+[motivation]: #motivation
+
+The current fee requirements suffer from being somewhat arbitrary, and not miner incentive compatible.
+They are not even linear; the fees required for the aggregate of two transactions is not necessarily equal to the sum
+of required fees.
+The current (and foreseeable) low price of Grin makes spamming the UTXO set rather cheaper than desired.
+Fee overpaying for higher priority to be included in full blocks fails when aggregated with minimal fee transactions.
+
+# Community-level explanation
+[community-level-explanation]: #community-level-explanation
+
+Grin constrains the size of blocks by a maximum block weight, which is a linear combination of the number of inputs, outputs, and kernels.
+When blocks fill up, miners are incentivized to pick transactions in decreasing order of fees per weight.
+Ideally then, a transaction that is more preferred by miners, is also more likely to be relayed.
+If a transaction A doesn't satisfy the minimum relay fees, while another transaction B does, but miners would rather include A,
+then A's publisher will have to transmit it directly to miners, which is undesirable.
+We therefore want the minimum relay fee be proportional to the weight of the transaction, minimizing arbitrariness.
+We can calibrate the fee system by stipulating that creating one output should cost at least one Grin-cent (formerly 0.4 Grin-cent).
+Linearity is another desirable property, which for instance allows the two parties in a payjoin to each pay their own input and output fees,
+while splitting the kernel fee.
+The least significant bits will specify a minimum fee overpayment factor, preventing aggregation with lesser overpaying transactions.
+
+# Reference-level explanation
```

Comment:
Currently this section reads as a single paragraph. 

It would benefit from bullet points or numbered list formatting of each specification requirement, like so:

```
* Bullet 1
* Bullet 2 
* Bullet 3
```

```
1. Numbered list item 1
1. Numbered list item 2
1. Numbered list item 3
```